src/oc_erchef/raml-docs/organizations/search.yaml

```diff
@@ -1,8 +1,108 @@
+description: |
+  A list of the indexes that can be searched. The indices include
```

Comment:
I think this might be missing context? What endpoint provides the list of indices?